### DISCUSSION TITLE
PEP735 support

### DISCUSSION THREAD

**Discussion by baggiponte** [pdm-project/pdm#2579] [General]
Hey there! I am sure PDM will support PEP735. What do you think of a hatch-like feature to manage different dev-dependencies groups in different envs?

I don't know if this makes sense and I may be wrong on this one, but I use libraries e.g mkdocs and ruff in dev dependencies and I wouldn't want these libraries' version constraints to impact the core dependencies.

**Comment by frostming** [comment]
> What do you think of a hatch-like feature to manage different dev-dependencies groups in different envs?

Not likely in the core repo but rather in a plugin, since it seems a duplicate work to me.

**Reply by baggiponte** [reply]
I understand your concern about duplicate work. I think PDM has an advantage position since it's the most feature-complete python package/project manager that complies with PEPs. This would ensure interoperability with package managers and CI/CD environments using `pip` to install dev dependencies (e.g. to run tests). I could help you with the duplication work, if you like.
